packages/message-kit/src/helpers/types.ts

```diff
@@ -36,14 +36,15 @@ export type ApiResponse = {
 
 export type CommandHandler = (
   context: HandlerContext,
-) => Promise<string | void | ApiResponse>;
+) => Promise<void | ApiResponse>;
 
 export type Handler = (context: HandlerContext) => Promise<void>;
 
 export type Config = {
   client?: ClientOptions;
   privateKey?: string;
   commandsConfigPath?: string;
+  logging?: "debug" | "info" | "warn" | "error" | "off";
```

Comment:
you can use `ClientOptions["logging"]` here instead so that it's always in sync with the option in the SDK